apis/apps/v1alpha1/configconstraint_conversion.go

```diff
@@ -19,129 +19,13 @@ package v1alpha1
 import (
 	"errors"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-
-	appsv1beta1 "github.com/apecloud/kubeblocks/apis/apps/v1beta1"
-	"github.com/apecloud/kubeblocks/pkg/constant"
 )
 
-var logger = logf.Log.WithName("application-resource")
-
 func (cc *ConfigConstraint) ConvertTo(dstRaw conversion.Hub) error {
-	logger.Info("Conversion Webhook: from v1alpha1 to v1", "name", cc.Name)
-	ccv1, ok := dstRaw.(*appsv1beta1.ConfigConstraint)
-	if !ok {
-		return errors.New("invalid destination object")
-	}
-	return convertToImpl(cc, ccv1)
+	return errors.New("not implemented")
 }
 
 func (cc *ConfigConstraint) ConvertFrom(srcRaw conversion.Hub) error {
-	ccv1, ok := srcRaw.(*appsv1beta1.ConfigConstraint)
```

Comment:
This will be invoked by cmd/helmhook/hook/multiversion/configconstraint.
Don't us need to keep this logic?

Reply:
Add these conversion logic back to the helm-hook.